### PR TITLE
implement countdown offset to compensate shutter-delay and cheese time

### PR DESCRIPTION
**Pull request by Metropo**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/andi34/photobooth/blob/dev/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "x" next to an item)

- [x] New feature

#### What changes did you make? (Give an overview)

I noticed, that some users that wanted to take pictures with the photobooth seemed to be confused or misled on the time the picture is taken. They assumed that at the moment the box displays "Cheeeeeese" and the live preview hides the image gets taken. This is probably encouraged by the mirror of the DSLR going down between live view and taking the picture.
A colleague suggested changing the countdown in a way, that it reaches 0 at the approximate time the photo is really taken. I implemented it using an offset that is added to the displayed countdown. It looks like this now
6, 5, 4, 3, Cheeeeeese [1 second] + shutter delay + Autofocus [1-2 seconds]
In my opinion, it is more intuitive this way. I would be glad to hear your opinion.

### CODE REVIEW

**Comment by andi34** [comment]
Thanks! I'll take a closer look at it later today. Changes to the language files only need to be done to the english file, all other languages need to be translated at crowdin once submitted ( https://crowdin.com/project/photobooth ).

**Comment by Metropo** [comment]
@andi34 Thanks for the information. I reverted the change to the german langauge file.

File: resources/lang/en.json [hunk 1]
```diff
@@ -415,6 +416,7 @@
   "pictures:picture_allow_delete": "Allow deletion of the image",
   "pictures:picture_cheese_time": "Cheeeeeeeese!-Timer:",
   "pictures:picture_cntdwn_time": "Countdown timer:",
+  "pictures:picture_cntdwn_offset": "Countdown offset:",
```
Comment (andi34) [inline resources/lang/en.json:418]: One Line up

File: resources/lang/en.json [hunk 2]
```diff
@@ -311,6 +311,7 @@
   "manual:pictures:picture_allow_delete": "If enabled pictures can be deleted on result page directly after they have been taken.",
   "manual:pictures:picture_cheese_time": "Set a time to display \"Cheeeeeeeese!\" after the countdown.",
   "manual:pictures:picture_cntdwn_time": "Set your countdown time.",
+  "manual:pictures:picture_cntdwn_offset": "Set an offset to the contdown to compensate for cheese and or focus and shutter delay.",
```
Comment (andi34) [inline resources/lang/en.json]: One line up

Comment (andi34) [inline resources/lang/en.json]: countdown not contdown

Comment (andi34) [reply resources/lang/en.json]: Also `and / or`

File: resources/lang/en.json (hunk 1 above)
Comment (Metropo) [reply resources/lang/en.json:418]: I guess it is alphabetically sorted?

File: resources/lang/en.json (hunk 2 above)
Comment (Metropo) [reply resources/lang/en.json]: What about: "Set an offset to the countdown to compensate for cheese time, focus and shutter delay." ?

File: resources/lang/en.json (hunk 1 above)
Comment (andi34) [reply resources/lang/en.json:418]: Right

File: resources/lang/en.json (hunk 2 above)
Comment (andi34) [reply resources/lang/en.json]: `Set an offset to the countdown to compensate the delay because of cheese time, focus and shutter speed.`

Maybe?

Comment (Metropo) [reply resources/lang/en.json]: perfekt :)

**Comment by Metropo** [comment]
I fixed the issues and rebased onto current dev branch.

**Pull request merged**